src/buildApplication/vercelConfig.ts

```diff
@@ -0,0 +1,47 @@
+import { readFile } from 'fs/promises';
+
+const supportedConfigVersion = 3;
+
+/**
+ * In src we don't make use of this object (just pass it along to the worker)
+ * for the full types of this object see: `templates/_worker.js/types.d.ts`
```

Comment:
I think the `types.d.ts` file is used to provide type safety in the `templates/_worker.js/index.ts` file, we do have some logic there so I think that it is quite helpful to have some type safety when we need to touch that file, don't you think?

Regarding `in defining and exporting this` you mean the type in this file?

We do need to access its version, so I guess we'd want to specify a type (unless we want to use `unknown` and casting?)

Would something like this be preferrable?
```diff
export type VercelConfig = {
	version: typeof supportedConfigVersion;
-	routes?: unknown;
-	images?: unknown;
-	wildcard?: unknown;
-	overrides?: unknown;
-	cache?: string[];
-	crons?: unknown;
};
```
(this would work but I fear that it might be a bit misleading and give the impression that the version is the only thing included in the file?)

___

Also I wanted to add that I did put there a todo for adding validation so I guess later we might want the correct type here.